.gitignore

```diff
@@ -1,7 +1,7 @@
 node_modules/
 
 .env
-
+package-lock.json
```

Comment:
Hi Kevin, package-lock.json should actually be included in source control, per: https://docs.npmjs.com/files/package-lock.json